### ISSUE TITLE
[Bug] Tabs tray FAB is added after the list of tabs in a11y tree

### ISSUE THREAD

**Issue by mcarare** [mozilla-mobile/fenix#20024]
Follow-up to https://github.com/mozilla-mobile/fenix/pull/19968

With FAB being shown when a11y is enabled, we should make sure is easily reachable.

## Steps to reproduce

Turn on accessibility mode.
Open the tabs tray.
Swipe to go through selectable views.

### Expected behavior

FAB should be traversed before the list of tabs

### Actual behavior

FAB is traversed after the list of tabs

### Device information

* Device vendor / model and Android version:  All
* Firefox for Android version:latest master

**Comment by LaurentiuApahideanSV** [comment]
Verified as fixed on Firefox Preview Nightly 6/17 in both normal and private browsing.
### Devices used:
- Google Pixel 3XL (Android 9)
- OnePlus 6T (Android 9)
